pkg/store/bucket.go

```diff
@@ -2661,79 +2663,79 @@ type queryStats struct {
 	blocksQueried int
 
 	postingsTouched          int
-	postingsTouchedSizeSum   int
+	PostingsTouchedSizeSum   units.Base2Bytes
```

Comment:
Nit: do these fields need to become public? Seems like an unintended change :thinking:

Reply:
It is intended change.  The fmt.Printf("%+v", stats) will print a number instead of human-readable for time.Duration If it is unexported. Here is a example in go-playgroud. https://go.dev/play/p/nL6EhzcEwkg
![image](https://user-images.githubusercontent.com/7621241/143459284-039ef96f-cf25-4ec2-b6ac-75424fd61cf7.png)

Reply:
Yes, I think this is the typical `fmt` 'gotcha' with unexported struct fields, see last paragraph in https://pkg.go.dev/fmt#hdr-Printing. AFAIK there's not other workaround other than exporting them.

Reply:
👍 
The doc says: 
> When printing a struct, fmt cannot and therefore does not invoke formatting methods such as Error or String on unexported fields.